### PR TITLE
Fix gettext catalog.

### PR DESCRIPTION
**Pull request by mbaechtold**

### CODE REVIEW

File: egov/contactdirectory/locales/de/LC_MESSAGES/egov.contactdirectory.po [hunk 1]
```diff
@@ -139,22 +167,24 @@ msgid "label_country"
...
 msgid "label_department"
 msgstr "Abteilung"
 
-#. Default: "E-Mail"
+#. Default: "Email"
+#: ./egov/contactdirectory/browser/contact.pt:93
 #: ./egov/contactdirectory/browser/contactfolder.pt:62
 #: ./egov/contactdirectory/content/contact.py:122
-#: ./egov/contactdirectory/content/member.py:129
+#, fuzzy
```
Comment (jone) [inline egov/contactdirectory/locales/de/LC_MESSAGES/egov.contactdirectory.po]: ❗️

File: egov/contactdirectory/locales/de/LC_MESSAGES/egov.contactdirectory.po [hunk 2]
```diff
@@ -139,22 +167,24 @@ msgid "label_country"
...
 msgid "label_email"
 msgstr "E-Mail"
 
-#. Default: "Mobile number"
-#: ./egov/contactdirectory/browser/contact.pt:73
+#. Default: "Fax"
+#: ./egov/contactdirectory/browser/contact.pt:86
 #: ./egov/contactdirectory/content/contact.py:141
 #: ./egov/contactdirectory/content/member.py:123
+#, fuzzy
```
Comment (jone) [inline egov/contactdirectory/locales/de/LC_MESSAGES/egov.contactdirectory.po]: ❗️

File: egov/contactdirectory/locales/en/LC_MESSAGES/egov.contactdirectory.po [hunk 3]
```diff
@@ -96,8 +124,8 @@ msgstr ""
 msgid "help_show_title"
 msgstr "Show Title of member contact"
 
-#. Default: "Academic_title"
-#: ./egov/contactdirectory/browser/contact.pt:41
+#. Default: "Academic title"
+#: ./egov/contactdirectory/browser/contact.pt:42
 #: ./egov/contactdirectory/content/contact.py:161
 #, fuzzy
```
Comment (jone) [inline egov/contactdirectory/locales/en/LC_MESSAGES/egov.contactdirectory.po]: ❗️

File: egov/contactdirectory/locales/de/LC_MESSAGES/egov.contactdirectory.po (hunk 2 above)
Comment (mbaechtold) [inline egov/contactdirectory/locales/de/LC_MESSAGES/egov.contactdirectory.po]: Will be removed in a later commit of this pull request.

**Comment by mbaechtold** [comment]
@jone, it's ready now.

**Pull request merged**